go/templates/index.tmpl

```diff
@@ -234,38 +228,16 @@
     if (products.includes('assets')) {
       $('#assets').show();
     }
-
-    var linkHandlerCommonOptions = {
-      apiVersion: 'v2',
-      clientName: 'Plaid Quickstart',
-      env: '{{ .plaid_environment }}',
-      product: products,
-      key: '{{ .plaid_public_key }}',
-      countryCodes: '{{ .plaid_country_codes }}'.split(','),
-    };
-    var oauthRedirectUri = '{{ .plaid_oauth_redirect_uri }}';
-    if (oauthRedirectUri != '') {
-      linkHandlerCommonOptions.oauthRedirectUri = oauthRedirectUri;
-    }
-    var oauthNonce = '{{ .plaid_oauth_nonce }}';
-    if (oauthNonce != '') {
-      linkHandlerCommonOptions.oauthNonce = oauthNonce;
-    }
     // This functionality is only relevant for the UK Payment Initiation product.
     if (products.includes('payment_initiation')) {
       $('.payment_initiation').show();
-      $.post('/set_payment_token', {}, function(data) {
-        console.log(data);
-        var paymentToken = data.payment_token;
-        $('#payment_token').text(paymentToken);
-
+      $.post('/create_link_token_for_payment', {}, function(data) {
+        localStorage.setItem('link_token', data.link_token);
         // In the case of payment_initiation product, we need to wait for
-        // payment token to be generated before the Link handler can be
-        // initialized.
+        // link token (with associated payment token) to be generated before the
```

Comment:
nit: `link_token configured for payment initiation` instead of `link token (with associated payment token)`